locales/en.json

```diff
@@ -340,7 +340,7 @@
     "errorMessage": "Something went wrong .. {errorMessage}",
     "errorMessageCannotBeCalculated": "Maximum amount cannot be calculated at this time.",
     "errorMessageTransferInvalid": "Could not transfer Circles as it is invalid.",
-    "errorMessageTransferNotFound": "Could not transfer Circles as there is currently no complete trust path between @{username} and you ..",
+    "errorMessageTransferNotFound": "Could not transfer Circles as the max amount you can send has changed due to the current implementation of the pathfinder. Try lower amounts.",
```

Comment:
@JacqueGM thanks, Looks good to me. Only one msg i will change. Because we are actually now doing a change n the pathfinder to make this better, so it should sound like this. 
Here: 

"errorMessageTransferNotFound": "The transaction could not be effected due to the trust path. Try smaller amounts."

Reply:
changed :)